docs/source/release_notes/release-notes.rst

```diff
@@ -48,6 +48,10 @@ develop
          - Fixed an issue that could cause AtlasConsole to print unnecessary amounts of input when commands were run.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2130>`__)
 
+    *    - |new|
+         - TransactionManagers exposes a method in which it is possible to specify the user agent to be used.
```

Comment:
nit: explicitly specify (or something along those lines)? It was always possible to, just not explicitly. I'm not sure what the best word for this is though.

(If we can't think of anything right now that's fine - the Atlas team can discuss this when finalising the release notes for 0.49)